certified-connectors/Witivio/apiProperties.json

```diff
@@ -1,25 +1,47 @@
 {
   "properties": {
-    "capabilities": [],
     "connectionParameters": {
       "token": {
+        "type": "oauthSetting",
         "oAuthSettings": {
-        },
-        "type": "oauthSetting"
+          "identityProvider": "aad",
+          "clientId": "9275fab6-86e0-4fb6-affa-1473100e8f22",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "https://witivio.com/internalapi"
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://witivio.com/internalapi"
+            }
+          }
+        }
       },
       "token:TenantId": {
+        "type": "string",
         "metadata": {
           "sourceType": "AzureActiveDirectoryTenant"
         },
-        "type": "string",
         "uiDefinition": {
           "constraints": {
-            "hidden": "true",
-            "required": "false"
+            "required": "false",
+            "hidden": "true"
           }
         }
       }
     },
-    "iconBrandColor": "#8F3163"
+    "iconBrandColor": "#007ee5",
```

Comment:
@tdechanterac The iconBrandColor should be `"#8F3163"` as per the submission in ISV studio.